lib/active_merchant/billing/gateways/d_local.rb

```diff
@@ -102,7 +102,13 @@ def lookup_country_code(country_field)
       def add_payer(post, card, options)
         address = options[:billing_address] || options[:address]
         post[:payer] = {}
-        post[:payer][:name] = card.name
+        if card.is_a?(CreditCard)
+          post[:payer][:name] = card.name
+        elsif card.is_a?(PspTokenizedCard)
+          post[:payer][:name] = options[:name]
```

Comment:
is name the document here?

Reply:
No @inai-shreyas, the document is added here - https://github.com/inaitech/active_merchant/pull/2/files#diff-200727fc728e189315a6fd1157c4f15a46ac7552f8fc88ee4f0aa57568164f8fR115.
This is the name of the end-user.